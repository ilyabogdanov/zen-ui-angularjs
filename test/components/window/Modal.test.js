const div = document.createElement("div");
document.body.appendChild(div);

describe("directive: zen-modal", function() {
    let element, scope;
    beforeEach(angular.mock.module("ZenUI"));
    beforeEach(inject(function($compile, $rootScope) {
        scope = $rootScope.$new();
        element = "<zen-modal></zen-modal>";
        element = $compile(element)(scope);
        scope.$digest();
        angular.element(div).append(element);
    }));
    afterEach(function () {
        while (div.firstChild) {
            div.removeChild(div.firstChild);
        }
    });

    it("should render", function () {
        expect(div.childNodes).toHaveLength(1);
        expect(div.firstChild).toMatchSnapshot();
    });
});
