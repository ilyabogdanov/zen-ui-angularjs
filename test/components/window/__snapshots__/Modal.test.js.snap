// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`directive: zen-modal should render 1`] = `
<div
  class="zen_ui__modal ng-scope"
>
  <div>
    <div>
      <div
        ng-transclude=""
      />
    </div>
  </div>
</div>
`;
